### ISSUE TITLE
FIX THE TRASH BS SYSTEM

### ISSUE THREAD

**Issue by fixthebssys** [ValveSoftware/Dota2-Gameplay#13773]
### Description

ASK NICELY BUT CUNTS DONT EVEN FIX YOUR BS SYSTEM

DONT FUKING IMPLEMENT SOMETHING THAT DOESN'T HELP 

YOU FUKING TRASH

PS dont delete my fucking post because you are incompetent at executing simple system

GO back to university before you attempt to implement system like a special need person

**Comment by arturkrugol** [comment]
they recieved money from Int and afk

**Comment by kriskorona** [comment]
Angry people have low behaviour score, working as intended

**Comment by fixthebssys** [comment]
you comment makes no sense .

**Comment by andraip** [comment]
Yeah Valve. Fix the BS system and finally perma ban unhinged people like OP.
